lib/seahorn/BvOpSem2MemRepr.cc

```diff
@@ -152,7 +156,15 @@ Expr OpSemMemLambdaRepr::MemCpy(Expr dPtr, Expr sPtr, unsigned len,
       Expr decl = bind::fname(addr);
       res = mk<LAMBDA>(decl, ite);
       LOG("opsem.lambda", errs() << "MemCpy " << *res << "\n");
+    } else {
+      // result is the same as source
+      res = memTrsfrRead;
     }
+  } else {
+    LOG("opsem.lambda", errs() << "Word size and pointer are not aligned and "
+                                  "alignment is not ignored!"
+                               << "\n");
+    assert(false);
```

Comment:
what happens in release mode when `assert(false)` is ignored?

Reply:
We return expr() in release mode.